ModelBundle/DependencyInjection/OpenOrchestraModelExtension.php

```diff
@@ -32,9 +32,6 @@ public function load(array $configs, ContainerBuilder $container)
                             ->setFactoryService('doctrine.odm.mongodb.document_manager')
                             ->setFactoryMethod('getRepository')
                             ->addArgument($content['class'])
-                            ->addMethodCall('setCurrentSiteManager', array(
```

Comment:
you can go a little further and suppress the current_site configuration parameter